sdk/communication/Azure.Communication.CallAutomation/tests/CallMedias/CallMediaTests.cs

```diff
@@ -319,5 +328,9 @@ public void MediaOperations_Return404NotFound(Func<CallMedia, Response<StartReco
                 }
             };
         }
+        private static void TestWrongTestSource_PlayOperations()
+        {
+            TextSource _wrongTextSource = new TextSource("PlayTTS test text.", "en-US-ElizabethNeural", GenderType.Female);
```

Comment:
Can you add invalid source locale such as hi-US (hindi-US) or ar-US (arabic-US) and check if this is caught by culture-info? 

Just for your information: Here is the list of valid source locale supported by cognitive services : https://learn.microsoft.com/en-us/azure/cognitive-services/speech-service/language-support?tabs=stt

Reply:
I just want to check if this is something CultureInfo can validate?  This is not a PR blocker.

Reply:
ok, will try to add some more cases